development/extensions/new_in_rhea.rst

```diff
@@ -98,15 +98,13 @@ Fortunately, the integration is pretty seamless and most existing extensions tha
   * ``generate_text_for_edit()``
   * ``generate_text_for_storage()``
   * ``strip_bbcode()``
-  * ``smiley_text()``
```

Comment:
Because the thing says "For example, the following phpBB functions will continue to work as they did in phpBB 3.1" which is misleading. People may think that `smiley_text()` can be used on 3.2 content. The other functions in that list can, but this one only works on 3.1.